src/components/forms/AcademicGrants2023Form.tsx

```diff
@@ -658,11 +658,13 @@ export const AcademicGrants2023Form: FC = () => {
             )}
           </FormControl>
 
-          <FormControl id='website-control' mb={8}>
+          <FormControl id='website-control' isRequired mb={8}>
             <FormLabel htmlFor='website' mb={1}>
-              <PageText fontSize='input'>Grant Proposal URL</PageText>
+              <PageText display='inline' fontSize='input'>
+                Grant Proposal URL
+              </PageText>
             </FormLabel>
-            <PageText fontSize='input' position='absolute' bottom='15.5px' left={4} zIndex={9}>
+            <PageText fontSize='input' position='absolute' top='80.5px' left={4} zIndex={9}>
```

Comment:
I think this broke some things on mobile:

<img width="364" alt="image" src="https://user-images.githubusercontent.com/54227730/217645728-682f594c-c3ad-490b-8c2d-8ce818d63c97.png">

Reply:
Also noting we aren't doing any validation that the string is actually in a URL format, not sure if we're concerned about this or not.

Reply:
Oops, I guess I'll need to invest more time on those styles.

Regarding the validation aspects, its a good point. I think its an aspect we need to fix in all the forms and fields we have. I already have an idea to fix this by using schemas with zod. But this will come later.

As of now, I think we are fine here as this field is not being used for other things.